.npmignore

```diff
@@ -27,6 +27,8 @@ docs/nav.yml
 docs/config.json
 docs/dockhand.js
 docs/template.html
+docs/package.json
+docs/content/
```

Comment:
`help-search.js` uses these files, we really do need this in the published module.

Reply:
Will the `node_modules` in this workspace be omitted from the pack?

Reply:
good catch!